tools/clang/test/CodeGenDXIL/hlsl/linalg/vector-matrix-proposal.hlsl

```diff
@@ -0,0 +1,143 @@
+// RUN: %dxc -I %hlsl_headers -T lib_6_9 -enable-16bit-types %s
```

Comment:
These are the code samples that appear in the proposal.  Do you think they should stay, since they're not really testing anything?

Reply:
I do think they should stay, yes.

Reply:
Though perhaps their location could be improved. I'm thinking DXC/examples/LinAlg/vector-matrix-proposal.hlsl